### ISSUE TITLE
ValueError: index must be monotonic

### ISSUE THREAD

**Issue by jasonebox** [GEUS-PROMICE/PROMICE-AWS-toolbox#3]
```python
runfile('/Users/jason/Dropbox/AWS/PROMICE/PROMICE-AWS-toolbox/PROMICE_toolbox/PROMICE_toolbox.py', wdir='/Users/jason/Dropbox/AWS/PROMICE/PROMICE-AWS-toolbox/PROMICE_toolbox')

runcell(0, '/Users/jason/Dropbox/AWS/PROMICE/PROMICE-AWS-toolbox/example.py')
figures and output folders already exist
Traceback (most recent call last):

  File "/Users/jason/Dropbox/AWS/PROMICE/PROMICE-AWS-toolbox/example.py", line 85, in <module>
    df_v4 = ptb.combine_hs_dpt(df_v4, site)

  File "/Users/jason/Dropbox/AWS/PROMICE/PROMICE-AWS-toolbox/PROMICE_toolbox/PROMICE_toolbox.py", line 294, in combine_hs_dpt
    smoothed_PT =  df['DepthPressureTransducer_Cor(m)'].interpolate(limit=72).rolling('7D', min_periods=1).mean().shift(-84, freq='h')

  File "/Users/jeb/anaconda3/envs/era5/lib/python3.8/site-packages/pandas/core/generic.py", line 10543, in rolling
    return Rolling(

  File "/Users/jeb/anaconda3/envs/era5/lib/python3.8/site-packages/pandas/core/window/rolling.py", line 169, in __init__
    self.validate()

  File "/Users/jeb/anaconda3/envs/era5/lib/python3.8/site-packages/pandas/core/window/rolling.py", line 1940, in validate
    self._validate_monotonic()

  File "/Users/jeb/anaconda3/envs/era5/lib/python3.8/site-packages/pandas/core/window/rolling.py", line 1980, in _validate_monotonic
    raise ValueError(f"{formatted} must be monotonic")

ValueError: index must be monotonic
```

Hey Baptiste,

I spent some time early this morning installing and working with https://github.com/GEUS-PROMICE/PROMICE-AWS-toolbox

I think I get it apart from some issues [see below] and so far I don't have the "adjusted" output, in particular the ice ablation record.

For Ken's TMB study, calibrating my reconstructed SMB, could you share the hourly "adjusted" output, or at least a timestamped ice ablation record for in particular these stations?

Alternatively, I could do a screen share and we ID whatever issues are preventing this output?

PROMICE_stations = [('KAN_L',(67.0955,-35.9748), 670), #Height sensor boom unusable, Height stake do not capture winter accumulation
                    ('KPC_L',(79.9108,-24.0828), 370),
                    ('KPC_U',(79.8347,-25.1662), 870), # pressure transducer not working
                    ('NUK_L',(64.4822,-49.5358), 530),
                    ('NUK_U',(64.5108,-49.2692), 1120),
                    ('QAS_L',(61.0308,-46.8493), 280),
                    ('QAS_M',(61.0998,-46.8330), 630),
                    ('QAS_U',(61.1753,-46.8195), 900),
                    ('SCO_U',(72.3933,-27.2333), 970),
                    ('TAS_A',(65.7790,-38.8995), 890),
                    ('TAS_L',(65.6402,-38.8987), 250),
                    ('THU_L',(76.3998,-68.2665), 570),
                    ('THU_U',(76.4197,-68.1463), 760),
                    ('UPE_L',(72.8932,-54.2955), 220),
                    ('UPE_U',(72.8878,-53.5783), 940)
                   ]

thx,

Jason
[example _JEBpy..txt](https://github.com/GEUS-PROMICE/PROMICE-AWS-toolbox/files/6120828/example._JEBpy.txt)

**Comment by BaptisteVandecrux** [comment]
Hi Jason,

I have just fixed the bug in the latest version. You can update your version of the repo.

You can use the updated [example.py](example.py) as you did previously, selecting QAS_L:
```
PROMICE_stations = [
                    ('QAS_L',(61.0308,-46.8493), 280)
                   ]
```

It should now run better.

Note that the markdown report only works if keeping the relative path of the figures intact.
The folder should then be structured as:
PROMICE-AWS-toolbox/out/figures
PROMICE-AWS-toolbox/out/Report.md
PROMICE-AWS-toolbox/out/Report_toc.md
In other words, the report only works if located in the same folder as the "figures" folder.

I am also sending you the processed data to save time.
